front/components/agent_builder/AgentBuilder.tsx

```diff
@@ -10,9 +13,17 @@ import {
 import { AgentBuilderLayout } from "@app/components/agent_builder/AgentBuilderLayout";
 import { AgentBuilderLeftPanel } from "@app/components/agent_builder/AgentBuilderLeftPanel";
 import { AgentBuilderRightPanel } from "@app/components/agent_builder/AgentBuilderRightPanel";
+import { submitAgentBuilderForm } from "@app/components/agent_builder/submitAgentBuilderForm";
+import { appLayoutBack } from "@app/components/sparkle/AppContentLayout";
+import logger from "@app/logger/logger";
 import { GPT_4O_MODEL_ID } from "@app/types";
 
 export default function AgentBuilder() {
+  const { owner } = useAgentBuilderContext();
+  const [isSaving, setIsSaving] = useState(false);
+  const router = useRouter();
```

Comment:
No need for that, `form` will give you a `isSubmitting`

Reply:
Handy!!